migration/migrator/main.py

```diff
@@ -81,9 +81,12 @@ def status(args):
                     continue
                 print_status(database, environment, loop_args)
                 database.close()
-            except OperationalError:
+            except OperationalError as exc:
                 print(
-                    'Could not get database for migrations for {}'.format(environment)
+                    'Could not get database for migrations for {}:\n  {}'.format(
+                        environment,
+                        str(exc).split("\n")[0]
```

Comment:
Is there a reason only the first line of the stack trace is printed? Wouldn't it be more helpful for debugging to show everything?

Reply:
sqlachemy produces the following string for the exception:
> (psycopg2.OperationalError) FATAL:  password authentication failed for user "submitty_dbuser1"
>
> (Background on this error at: http://sqlalche.me/e/13/e3q8)

I want to hide the final line as the link is not useful (and perhaps confusing) to a non-developer familiar with sqlachemy. I do want the first line though as it gives the specific error (the `(psycopg2.OperationalError)` bit do not necessarily care for, but not strongly enough to strip a strip method for it.